shared/chat/conversation/header/index.desktop.js

```diff
@@ -76,4 +78,12 @@ const styleLeft = {
   marginLeft: globalMargins.tiny,
 }
 
+const shhIconStyle = {
+  color: globalColors.black_20,
+  fontSize: 20,
+  marginLeft: globalMargins.xtiny,
+  top: globalMargins.tiny,
+  position: 'absolute',
```

Comment:
Hmm...I think `position: 'absolute'` isn't what you want here, as it depends on how Icon is implemented. I don't know enough about CSS to know the right thing to do here, though... @MarcoPolo do you know?

Reply:
I don't understand what you mean about how Icon is implemented?

Reply:
From chat with Marco: "I meant that passing in `position: absolute` in the style for the Icon relies on Icon being implemented as a box containing the span with the style"

Reply:
Ok, Marco and I tweaked it. Can you take a fresh look?